### ISSUE TITLE
 ✨ Gathering requires for Access to QuickSight for Non MoJ staff

### ISSUE THREAD

**Issue by jacobwoffenden** [ministryofjustice/analytical-platform#4254]
### Describe the feature request.

This feature request supersedes https://github.com/ministryofjustice/data-platform-support/issues/575

### Describe the context.

https://github.com/ministryofjustice/data-platform-support/issues/575

### Value / Purpose

https://github.com/ministryofjustice/data-platform-support/issues/575

### User Types

Non MoJ staff

**Comment by vonbraunbates** [comment]
See follow-up comments [here](https://github.com/ministryofjustice/data-platform-support/issues/575).

Note that I am not sure that my request is a typical user need for non-MOJ staff:
- we (CT Police and MOD) have a lot of unusual security requirements
- the team aren't data consumers just viewing dashboards: we need them to Author the analyses first
- there are a lot of end-users (300-400) who might typically be Readers in QS parlance, but we haven't decided whether to roll this out to them yet
- our dataset needs some way for it to be **not viewable by everyone** with a QS account -- we'd like to grant access on an individual basis after proof of SC supplied
- we need something quickly (sorry, MOD always wants things done yesterday!) but we're open to it being temporary

Much as I like this feature request, I am not entirely sure it supercedes my earlier ticket (yet).

**Comment by gfowler-moj** [comment]
Hi Francesca. Is Nicola MW aware of this work, and have you engaged with her re: Information Assurance etc? There's presumably a data protection/sharing aspect to this too, but one of my chief concerns is that we're talking about opening up our platform and tools to users that aren't part of our organisation.

This raises some questions for me around commercial aspects (we pay for licenses for QS users), as well as security and how we can guarantee that we're managing users and data access robustly (if one of these users leaves MOD, how do we know to remove their access?).

We're going to have a chat today about this with security, but I just wondered how much of these sorts of discussions you may have already had (if at all)?

**Comment by vonbraunbates** [comment]
@gfowler-moj all valid points:
- Nicola MW is aware of the Hub: we've been filling in a lot of data governance forms via her team for this work.  There is also a formal information sharing agreement between the Hub's partner agencies (of which HMPPS is one, which is where we come in).
- I talked to Simon Heron about this project and have had some discussions with Tevide (Simon's replacement).  I haven't raised the issue with anyone outside of pandas cyber though.
- The central co-ordination team at the Hub would have to take responsibility for joiners, movers and leavers.  Happy to talk to them about adding whatever we'd need to their on- and off-boarding process.
- We haven't addressed commerical aspects yet.  Previously the plan was to use Apache Superset -- which would have meant no licence fees -- but on Tevide's advice we switched to QS on the AP.  How much would those licences cost per user?

Should we bring this up for discussion at the next security and assurance meeting?

(Let's see if @vonbraunbates  lists this in my relevant issues...I keep losing this URL.)
